docs/sources/dashboards/export-import.md

```diff
@@ -34,7 +34,7 @@ the dashboard, and will also be added as a required input when the dashboard is
 
 ## Import dashboard
 
-To import a dashboard click the + icon in the side menu, and then click **Import**.
+To import a dashboard click the **Dashboards** icon in the side menu, and then click **Import**.
```

Comment:
You don't actually click the dashboards icon, you hover it. 

Maybe this could be reworded to something more like "Click **Import** under **Dashboards** in the side menu", or something like that?